mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -382,8 +399,23 @@ pub fn atom_node(docker: &'_ Cli) -> DockerNode<'_> {
     };
     assert!(atom_node_state_dir.exists());
 
+    let atom_node_runtime_dir = {
+        let mut current_dir = std::env::current_dir().unwrap();
+        current_dir.pop();
+        current_dir.pop();
+        current_dir.join(".docker/container-runtime/atom-testnet-data")
+    };
+
+    // Remove runtime directory if it exists to copy container files to a clean directory
+    if atom_node_runtime_dir.exists() {
+        remove_dir_all(&atom_node_runtime_dir).unwrap();
+    }
+
+    // Copy container files to runtime directory
+    copy_dir_all(&atom_node_state_dir, &atom_node_runtime_dir).unwrap();
```

Comment:
Currently IBC directory is not handled. I think we should copy entire `.docker/container-state` into `.docker/container-runtime`  from https://github.com/KomodoPlatform/komodo-defi-framework/blob/aae70ee81911b39c8aa26f50e3b98373414531d6/mm2src/mm2_main/tests/docker_tests_main.rs#L45

so we don't need to handle inner directories individually from the container functions.

Reply:
Done